### PR TITLE
navbar support for email/password users

### PR DESCRIPTION
**Pull request by sleepylemur**

### CODE REVIEW

File: src/scripts/main-nav.js [hunk 1]
```diff
@@ -117,18 +117,27 @@
...
+  function getLoginElem (id, nickname, githubAvatar) {
+    if (nickname) {
+      // github user so show their nickname and avatar
+      var avatarImageURL = githubAvatar;
+      var label = nickname;
+    } else {
+      // email/password user so just show 'account' and default avatar
+      var avatarImageURL = 'common/styleguide/images/default-avatar.png';
+      var label = 'Account';
+    }
```
Comment (rfriberg) [inline src/scripts/main-nav.js]: You could even simplify this some:

```
    // email/password user so just show 'account' and default avatar
    var avatarImageURL = 'common/styleguide/images/default-avatar.png';
    var label = 'Account';

    if (nickname) {
      // github user so show their nickname and avatar
      avatarImageURL = githubAvatar;
      label = nickname;
    }
```
Is it possible for someone to have a `nickname` but no `githubAvatar`?

Comment (rfriberg) [reply src/scripts/main-nav.js]: Also, "Account" is totally fine. But do we have access to a first name when users register with an email?

**Review by rfriberg — APPROVED** [review]
👍

File: src/scripts/main-nav.js (hunk 1 above)
Comment (sleepylemur) [reply src/scripts/main-nav.js]: We don't have access to name, and I discussed with ekta whether it was viable to put the email address or some shortened form of it in the navbar and she preferred to go with 'Account' at least initially.

That simplification is actually the way I initially wrote it and then I thought maybe the if/else was clearer. I'll switch it back.

**Comment by souperneon** [comment]
add the word "Account" next to avatars for all accounts and remove the GitHub names

**Pull request merged**